include/ddc/kernels/splines/constant_extrapolation_rule.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "view.hpp"
+
+namespace ddc {
+template <class DDim>
```

Comment:
```suggestion
namespace ddc {

template <class DDim>
```
Please think of adding a new line when opening a namespace